dev/tools/gen_defaults/pubspec.yaml

```diff
@@ -7,34 +7,31 @@ environment:
 
 dependencies:
 
-  async: 2.8.2 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
-  charcode: 1.3.1 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
-  collection: 1.15.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
-  http_parser: 4.0.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
-  source_span: 1.8.1 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
-  string_scanner: 1.1.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
-  term_glyph: 1.2.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
-  typed_data: 1.3.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
```

Comment:
@darrenaustin looking at https://github.com/flutter/flutter/pull/95370 this change looks correct, but I'm not sure why these packages were included in `dependencies` in the first place.  Did you create this pubspec manually, or did you run `flutter update-packages` and `AUTOGENERATED` was copied and pasted from somewhere?

Reply:
This looks fine. The only dependencies this tool has is the two dev_dependencies `path` and `test`.

I was having a problem getting this pubspec into the tree. I wound up copying entries from gen_keycodes's pubspec to get it to pass the analyzer. Sorry this is causing confusion. What is the correct way to add a new pubspec to the tree? Thx.

Reply:
I guess what you did, and then an additional step of running `update-packages --force-upgrade` to match the other pubspec versions and clean up.